docs/known-issues.md

```diff
@@ -1 +1,25 @@
-Placeholder - see [the WIP known issues doc](https://brevi.link/unreal-gdk-known-issues).
\ No newline at end of file
+## Unreal GDK Known Issues
+
+Known issue = any major user-facing bug or lack of user-facing feature that:
+1. diverges from vanilla Unreal design or implementation (e.g. ordering of reliable RPCs), **OR**
+1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+
+| Issue | Date added | Ticket | Workaround? | Done? |
+|-------|-------------------|--------|-------------|-------|
+| Order of reliable RPCs is not respected in case of unresolved UObject* parameter. | 2018-06-14 | [UNR-363](https://improbableio.atlassian.net/browse/UNR-336) | Potential user-code workaround. | No|
+| Reliable multicast RPCs aren’t supported. | 2018-06-14 | | Ask yourself if you really need a reliable multicast RPC. | No |
+| Project build `PCHUsageMode` must be set to `UseExplicitOrSharedPCHs`. | 2018-06-27 |[UNR-354](https://improbableio.atlassian.net/browse/UNR-354)| | No |
+| Adaptive Unity builds are not supported. | 2018-07-28 | | Disable `bUseAdaptiveUnityBuild` following [Epic’s documentation](https://docs.unrealengine.com/en-US/Programming/UnrealBuildSystem/Configuration). | No |
+| When working on a blueprint class, we need to specify the blueprint class in the DefaultEditorSpatialGDk.ini. E.g SampleGameCharacterBP, but we actually need to add SampleGameCharacterBP_C, the _C is an Unreal workaround. | 2018-06-27 | [UNR-363](https://improbableio.atlassian.net/browse/UNR-363) | Append `_C` to blueprint classes names in the `.ini` file. | No |
```

Comment:
I don't understand this - could you clarify whether "we" should say "you" (as in, the user)? 

And the bit that says "but we actually need to..." is a bit confusing - it looks like we're saying you need to do one thing, and then that you actually need to do a different thing.

EDIT: I think I've just got it - would this be better?

> When working on a Blueprint class, we need to specify the Blueprint class in the `DefaultEditorSpatialGDk.ini` and add `_C` onto the end of it - for example, `SampleGameCharacterBP_C`. (The `_C` is an Unreal workaround.)

Reply:
As it turns out, this is no longer an issue, so we should remove the entry